src/Common/PlanetParameters/PlanetParameters.jl

```diff
@@ -39,6 +39,7 @@ using ..ParametersType
 @exportparameter cv_i              cp_i          "Isochoric specific heat ice (J/kg/K)"
 @exportparameter T_freeze          273.15        "Freezing point temperature (K)"
 @exportparameter T_min             150.0         "Minimum temperature guess in saturation adjustment (K)"
+@exportparameter ΔT_min            10.0          "Minimum interval for saturation adjustment guess"
```

Comment:
I'd leave this parameter in `MoistThermodynamics` since it'll only be used there. It's also quite large (something like 3 K would suffice, I'd think.)